### PR TITLE
[Localization] Missing items translations

### PR DESCRIPTION
**Pull request by Adri1**
## What are the changes the user will see?
Added missing items translations

## Why am I making these changes?
To have them translated

## Checklist
- [X] **I'm using `beta` as my base branch**
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?

### CODE REVIEW

File: src/locales/ko/modifier-type.json [hunk 1]
```diff
@@ -495,6 +499,22 @@
...
+    "RAZOR_FANG": "Razor Fang",
+    "REAPER_CLOTH": "Reaper Cloth",
+    "ELECTIRIZER": "Electirizer",
+    "MAGMARIZER": "Magmarizer",
+    "PROTECTOR": "Protector",
+    "SACHET": "Sachet",
+    "WHIPPED_DREAM": "Whipped Dream",
+    "LEADERS_CREST": "Leader's Crest",
+    "SUN_FLUTE": "Sun Flute",
+    "MOON_FLUTE": "Moon Flute",
```
Comment (CodeTappert) [inline src/locales/ko/modifier-type.json:516]:
```suggestion
"UPGRADE": "업그레이드",
    "DUBIOUS_DISC": "괴상한패치",
    "DRAGON_SCALE": "용의비늘",
    "PRISM_SCALE": "고운비늘",
    "RAZOR_CLAW": "예리한손톱",
    "RAZOR_FANG": "예리한이빨",
    "REAPER_CLOTH": "영계의천",
    "ELECTIRIZER": "에레키부스터",
    "MAGMARIZER": "마그마부스터",
    "PROTECTOR": "프로텍터",
    "SACHET": "향기주머니",
    "WHIPPED_DREAM": "휘핑크림",
    "LEADERS_CREST": "대장의징표",
    "SUN_FLUTE": "태양의피리",
    "MOON_FLUTE": "달의피리",
```

File: src/locales/ko/modifier-type.json [hunk 2]
```diff
@@ -495,6 +499,22 @@
     "TART_APPLE": "새콤한사과",
     "STRAWBERRY_SWEET": "딸기사탕공예",
     "UNREMARKABLE_TEACUP": "범작찻잔",
+"UPGRADE": "업그레이드",
+    "DUBIOUS_DISC": "괴상한패치",
```
Comment (CodeTappert) [inline src/locales/ko/modifier-type.json:503]:
```suggestion
    "UPGRADE": "업그레이드",
    "DUBIOUS_DISC": "괴상한패치",
```

File: src/locales/it/modifier-type.json [hunk 3]
```diff
@@ -495,6 +499,22 @@
...
+    "REAPER_CLOTH": "Reaper Cloth",
+    "ELECTIRIZER": "Electirizer",
+    "MAGMARIZER": "Magmarizer",
+    "PROTECTOR": "Protector",
+    "SACHET": "Sachet",
+    "WHIPPED_DREAM": "Whipped Dream",
+    "LEADERS_CREST": "Leader's Crest",
+    "SUN_FLUTE": "Sun Flute",
+    "MOON_FLUTE": "Moon Flute",
+
```
Comment (CodeTappert) [inline src/locales/it/modifier-type.json:517]:
```suggestion
    "UPGRADE": "Upgrade",
    "DUBIOUS_DISC": "Dubbiodisco",
    "DRAGON_SCALE": "Squama drago",
    "PRISM_SCALE": "Squama bella",
    "RAZOR_CLAW": "Affilartiglio",
    "RAZOR_FANG": "Affilodente",
    "REAPER_CLOTH": "Terrorpanno",
    "ELECTIRIZER": "Elettritore",
    "MAGMARIZER": "Magmatore",
    "PROTECTOR": "Copertura",
    "SACHET": "Bustina aromi",
    "WHIPPED_DREAM": "Dolcespuma",
    "LEADERS_CREST": "Simbolo del capo",
    "SUN_FLUTE": "Flauto solare",
    "MOON_FLUTE": "Flauto lunare",
    
```

File: src/locales/it/modifier-type.json [hunk 4]
```diff
@@ -358,6 +358,10 @@
       "name": "Fiammosfera",
       "description": "Sfera bizzarra che procura una scottatura a chi l’ha con sé in una lotta."
     },
+    "EVOLUTION_TRACKER_GIMMIGHOUL": {
+      "name": "Treasures",
+      "description": "This Pokémon loves treasure! Keep collecting treasure and something might happen!"
+    },
```
Comment (CodeTappert) [inline src/locales/it/modifier-type.json:364]:
```suggestion
    "EVOLUTION_TRACKER_GIMMIGHOUL": {
      "name": "보물",
      "description": "이 포켓몬은 보물을 좋아한다. 보물을 모으다 보면 어떤 일이 일어날 지도 모른다!"
    },
```

File: src/locales/ko/modifier-type.json [hunk 5]
```diff
@@ -358,6 +358,10 @@
       "name": "화염구슬",
       "description": "이 도구를 지닌 포켓몬은 턴이 끝나는 시점에 상태이상에 걸리지 않았다면 화상 상태가 된다."
     },
+    "EVOLUTION_TRACKER_GIMMIGHOUL": {
+      "name": "Treasures",
+      "description": "This Pokémon loves treasure! Keep collecting treasure and something might happen!"
+    },
```
Comment (CodeTappert) [inline src/locales/ko/modifier-type.json:364]:
```suggestion
    "EVOLUTION_TRACKER_GIMMIGHOUL": {
      "name": "보물",
      "description": "이 포켓몬은 보물을 좋아한다. 보물을 모으다 보면 어떤 일이 일어날 지도 모른다!"
    },
```

File: src/locales/es/modifier-type.json [hunk 6]
```diff
@@ -358,6 +358,10 @@
       "name": "Llamasfera",
       "description": "Extraña esfera que causa quemaduras a quien la usa en combate."
     },
+    "EVOLUTION_TRACKER_GIMMIGHOUL": { 
+      "name": "Treasures", 
+      "description": "This Pokémon loves treasure! Keep collecting treasure and something might happen!"
+    },
```
Comment (CodeTappert) [inline src/locales/es/modifier-type.json:364]:
```suggestion
    "EVOLUTION_TRACKER_GIMMIGHOUL": {
      "name": "Tesoros",
      "description": "¡A este Pokémon le encantan los tesoros! ¡Sigue coleccionandólos y tal vez pase algo!"
    },
```

File: src/locales/ja/modifier-type.json [hunk 7]
```diff
@@ -240,6 +240,8 @@
     "TOXIC_ORB": { "name": "どくどくだま", "description": "触ると　毒をだす　不思議な玉。\n持たせると　戦闘中に　猛毒の状態に　なる" },
     "FLAME_ORB": { "name": "かえんだま", "description": "触ると　熱をだす　不思議な玉。\n持たせると　戦闘中に　やけどの状態に　なる。" },
 
+    "EVOLUTION_TRACKER_GIMMIGHOUL": { "name": "Treasures", "description": "This Pokémon loves treasure! Keep collecting treasure and something might happen!"},
+
```
Comment (CodeTappert) [inline src/locales/ja/modifier-type.json:244]:
```suggestion
    "EVOLUTION_TRACKER_GIMMIGHOUL": {
      "name": "宝物",
      "description": "このポケモンは　宝物が　大好き！　宝物を　集め続けると　良いことが　起こる　かもしれません！"
    },

```

File: src/locales/zh_CN/modifier-type.json [hunk 8]
```diff
@@ -358,6 +358,10 @@
       "name": "火焰宝珠",
       "description": "触碰后会放出热量的神奇宝珠。\n携带后，在战斗时会变成灼伤状态。"
     },
+    "EVOLUTION_TRACKER_GIMMIGHOUL": {
+      "name": "Treasures",
+      "description": "This Pokémon loves treasure! Keep collecting treasure and something might happen!"
```
Comment (Tempo-anon) [inline src/locales/zh_CN/modifier-type.json]:
```suggestion
      "name": "金子宝物",
      "description": "这个小精靈爱金子! 继续挑金子然后谁知道什么会发生!"
```

File: src/locales/zh_TW/modifier-type.json [hunk 9]
```diff
@@ -358,6 +358,10 @@
       "name": "火焰寶珠",
       "description": "觸碰後會放出熱量的神奇寶珠。\n攜帶後，在戰鬥時會變成灼傷狀態。"
     },
+    "EVOLUTION_TRACKER_GIMMIGHOUL": {
+      "name": "金子宝物",
```
Comment (Tempo-anon) [inline src/locales/zh_TW/modifier-type.json:362]: `金子寶物`

宝 -> 寶

**Review by Tempo-anon — APPROVED** [review]

**Review by SangaraSorama — APPROVED** [review]

**Review by SangaraSorama — APPROVED** [review]

**Pull request merged**